prometheus-to-sd/main.go

```diff
@@ -144,32 +143,18 @@ func main() {
 		glog.Error(http.ListenAndServe(fmt.Sprintf("%s:%d", *debugAddress, *debugPort), expvar.Handler()))
 	}()
 
-	var client *http.Client
-
+	var options []option.ClientOption
 	if *gceTokenURL != "" {
```

Comment:
Do we need to handle `else` case?

Reply:
First, two `else`s before this PR were actually doing the same thing (ie. `google.DefaultClient()` creates google.DefaultTokenSource() and puts it in oauth2.NewClient)
Second, the projectOverride is not needed because everytime the client posts metrics, the request body contains the project name (see https://github.com/GoogleCloudPlatform/k8s-stackdriver/pull/742/files#diff-fabb01b5a0dd10f9e48b220cf31c93d51c6fd5a5516b4254ca63232c43a0dfe8R61)